circle.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
 
   override:
     - python setup.py install
+    - pip install -U scikit-learn
```

Comment:
Should this be added to the requirements.txt?

Reply:
I had added it to requirements.txt in commit [d9973b0](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/85/commits/d9973b008a013efae7f0cbb3c3afe836bd4ee2fc). If I'm not wrong, the dependencies in requirements.txt are installed according to the order in the file.
The CircleCI test for this commit failed because it was trying to install scikit-learn before scipy. I tried looking for explanations and solutions but this is the only working solution I could find then (I completely forgot about it later).

Reply:
As it turns out, installing scikit-learn causes problems as `setup` tries to do a simple `pip` install from the Python Package Index. A simple install fails if `numpy` and `scipy` aren't already installed on the system. 
Take a look at the installation page for scikit-learn : https://github.com/scikit-learn/scikit-learn/blob/master/doc/install.rst
Also, take a look at this issue and PR on scikit-learn's GitHub repo : 
https://github.com/scikit-learn/scikit-learn/issues/4164
https://github.com/scikit-learn/scikit-learn/pull/6990

Anaconda offers scikit-learn as a part of its distribution.
It may make sense to suggest installing Anaconda(or MiniConda) before running `setup.py`.